bundle/compliance/policy/aws_iam/data_adapter.rego

```diff
@@ -1,9 +1,43 @@
 package compliance.policy.aws_iam.data_adapter
 
+import data.compliance.lib.common
+import future.keywords.every
+
 is_pwd_policy {
 	input.subType == "aws-password-policy"
 }
 
+is_iam_user {
+	input.subType == "aws-iam-user"
+	not input.resource.name == "<root_account>"
+}
+
 pwd_policy = policy {
+	is_pwd_policy
 	policy := input.resource
 }
+
+iam_user = user {
+	is_iam_user
+	user := input.resource
+}
+
+used_active_access_keys = {access_key |
+	access_key = iam_user.access_keys[_]
+	access_key.active
+	access_key.has_used
+}
+
+unused_active_access_keys = {access_key |
+	access_key = iam_user.access_keys[_]
+	access_key.active == true
```

Comment:
nit: same as in `used_active_access_keys`, usually in rego we don't check for true explicitly
```suggestion
	access_key.active
```